esi_leap/common/flocx_market.py

```diff
@@ -0,0 +1,110 @@
+from esi_leap.common import statuses
+from esi_leap.db import api as dbapi
+from esi_leap.objects import flocx_market_client
+from esi_leap.objects import offer
+import json
+from keystoneauth1 import adapter
+from keystoneauth1 import loading as ks_loading
+from oslo_config import cfg
+import datetime
+
+CONF = cfg.CONF
+
+def encode_hook(pairs):
+    new_pairs = {}
+    for key, value in pairs.iteritems():
+        if isinstance(value, unicode):
+            value = value.encode('utf-8')
+        if isinstance(key, unicode):
+            key = key.encode('utf-8')
+        new_pairs[key] = value
+    return new_pairs
```

Comment:
Actually, I'm not sure you need this encode_hook at all; the code fails for me unless I take it out. Do you get an error without this?

Reply:
Oh, I find that it's because my virtual environment use python2. Now I change it to python3 and take the encode_hook out.